CloverSDK/Models/TransformTypes.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  TransformTypes.swift
+//  CloverSDK
+//
+//  Created by Yusuf on 12/24/15.
+//  Copyright © 2015 Clover Network, Inc. All rights reserved.
+//
+
+import SwiftyJSON
+import ObjectMapper
+
+public class CLVDateTransform: TransformType {
+  public typealias Object = NSDate
+  public typealias JSON = Double
+  
+  public init() {}
+  
+  public func transformFromJSON(value: AnyObject?) -> NSDate? {
+    if let timeInt = value as? Double {
```

Comment:
Would it be safer to use a Long for timestamps?  Or time_t? http://stackoverflow.com/a/2997262